network_modeling/models/cray-network-hardware.yaml

```diff
@@ -170,6 +171,15 @@ network_hardware:
       - count: 1
         speed: 1
         slot: "mgmt"
+  - name: "SLINGSHOT 200G 64P"
+    vendor: "cray"
+    model: "slingshot_hsn_switch"
+    type: "switch"
+    deprecated: True
+    preserve_hardware_model_port_layout: True
+    ports:
+      - count: 64
```

Comment:
ahaha.. here's the crux.  Are there really 64 BMC ports?
Two options here:
1. Track only the bmc ports.  In this case `count: 1`.  
2. Add 64 200G ports and a single 1G port.

CSM doesn't care about the 200G ports so I'd pick option 1 and not log the 200G ports.  Technically we want this to be a "mgmt" slot but that would require the SHCD to explicitly mark it in the Slot column.  Thus leave it w/o slot.